### DISCUSSION TITLE
Is there a way to reuse (parts of) a component?

### DISCUSSION THREAD

**Discussion by matthieupeeters** [sqlpage/SQLPage#284] [Q&A]
If I want to reuse or nest templates, there is **Handlebars.registerPartial** in handlebars. This doesn't seem to be supported by SQLPage. Is there a way to make code reuse possible, or should I use some kind of pre-processor?

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage!

Unfortunately, partials are not supported... yet. What is your use case?

**Reply by matthieupeeters** [reply]
Thanks for the welcome and the reply!

Sorry for the late reply, I am very busy lately. The idea is to reuse a (part of a) component to show different data-types in the same way in multiple templates. But it will take a while before I can demonstrate it, due to my time constraints.
